client/src/components/qa/AddAnswer.jsx

```diff
@@ -27,67 +27,69 @@ function AddAnswer({
-        { images.length < 5 && (
-          <input type="file" accept="image/*" onChange={handleImg} />
-        )}
-        <br />
-        {images.map((image) => (
-          <img
-            src={image}
-            alt="thumbnail"
-            className="ans-thumbnail"
-          />
-        ))}
-        <br /> <br />
-        <button type="submit" className="qa-add-submit">Submit</button>
-      </form>
-    </div>,
-  );
-
   return (
     <div>
-      <div className="qa-modal-shadow" onClick={close} />
-      <div className="qa-modal">
-        <div className="qa-modal-content">
-          <span onClick={close} className="close-button">&times;</span>
-          {content}
+      <div className={`qa-modal-shadow ${theme}`} onClick={close} />
+      <div className={`qa-modal ${theme}`}>
+        <div className={`qa-modal-content ${theme}`}>
+          <span onClick={close} className={`close-button ${theme}`}>&times;</span>
+          <div>
+            <h1 className="qa-add-title">Submit Your Answer </h1>
+            <h2>{name}: {body}</h2>
+            <form onSubmit={handleAddAnswer}>
+              <h3>Your Answer <span style={{ color: 'red' }}>*</span></h3>
+              <textarea
+                className="qa-add-input"
+                rows="5"
+                cols="40"
+                maxLength="1000"
+                onChange={(e) => { setAnswerInput(e.target.value); }}
+              />
+              <h3>What is your nickname? <span style={{ color: 'red' }}>*</span></h3>
+              <input
+                className="qa-add-name"
+                type="text"
+                maxLength="60"
+                placeholder="Example: jackson11!"
+                onChange={(e) => { setAnswerName(e.target.value); }}
+                required
+              />
+              <div className="qa-add-subtext">for privacy reasons, do not use your full name or email address</div>
+              <h3>Your Email <span style={{ color: 'red' }}>*</span></h3>
+              <input
+                className="qa-add-email"
+                type="email"
+                maxLength="60"
+                placeholder="Example: jack@email.com"
+                onChange={(e) => { setAnswerEmail(e.target.value); }}
+                required
+              />
+              <div className="qa-add-subtext">for authentication reasons, you will not be emailed</div>
+              <h3>Upload your photos</h3>
+              { images.length < 5 && (
+                <input type="file" accept="image/*" onChange={handleImg} />
+              )}
+              <br />
+              {images.map((image) => (
+                <img
+                  src={image}
+                  alt="thumbnail"
+                  className="ans-thumbnail"
+                />
+              ))}
+              <br /> <br />
+              <button type="submit" className="qa-add-submit">Submit</button>
+            </form>
+          </div>
         </div>
       </div>
     </div>
```

Comment:
Im not sure why this diff is so large. It doesn't seem like you made any vast changes. All I see is thematic styling.